src/Fleck/Interfaces/ISocket.cs

```diff
@@ -15,6 +15,7 @@ public interface ISocket
         int RemotePort { get; }
         Stream Stream { get; }
         bool NoDelay { get; set; }
+        bool RestartAfterListenError { get; set; }

```

Comment:
Because this property is only used by `WebSocketServer`, I would put it on that class.